### PR TITLE
Add examples for the condensation function

### PR DESCRIPTION
**Pull request by kpetridis24**
Added two examples for the better understanding of the condensation funciton use

### CODE REVIEW

**Comment by kpetridis24** [comment]
I fixed the new line issues and re-created the previously submitted PR with no issues in the current version.

File: networkx/algorithms/components/strongly_connected.py [hunk 1]
```diff
@@ -371,6 +371,26 @@ def condensation(G, scc=None):
     NetworkXNotImplemented
         If G is undirected.
 
+    Examples
+    -----
+    >>> G = nx.barbell_graph(4, 0)
+    >>> G.remove_edge(3, 4)
+    >>> G = nx.DiGraph(G)
```
Comment (MridulS) [inline networkx/algorithms/components/strongly_connected.py:378]: Thanks for this!
Could you also add a quick comment about the graph construction. I wrote down the example
quickly without a comment. Could you put something like `# create 2 distinct SCC using the barbell graph`

File: networkx/algorithms/components/strongly_connected.py [hunk 2]
```diff
@@ -371,6 +371,26 @@ def condensation(G, scc=None):
...
+    >>> H = nx.condensation(G)
+    >>> H.nodes.data()
+    NodeDataView({0: {'members': {0, 1, 2, 3}}, 1: {'members': {4, 5, 6, 7}}})
+    >>> H.graph['mapping']
+    {0: 0, 1: 0, 2: 0, 3: 0, 4: 1, 5: 1, 6: 1, 7: 1}
+
+    >>> G = nx.complete_graph(7)
+    >>> G = nx.DiGraph(G)
```
Comment (MridulS) [inline networkx/algorithms/components/strongly_connected.py:386]: There is no need to cast the `DiGraph`, for complete graphs we can directly get a `DiGraph` with `create_using`.
`G = nx.complete_graph(7, create_using=nx.DiGraph)` should do the job :)

Comment (kpetridis24) [reply networkx/algorithms/components/strongly_connected.py:386]: Oh nice, i haven't used this function ever. Thanks for the clarification.

**Review by MridulS — APPROVED** [review]

**Review by dschult — APPROVED** [review]

**Pull request merged**